<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuChange_Mode">
     <property name="title">
      <string>Change Mode</string>
     </property>
     <addaction name="actionPowder_TOF"/>
     <addaction name="actionPowder_Elastic"/>
     <addaction name="actionSingle_Crystal_Elastic"/>
    </widget>
    <addaction name="menuChange_Mode"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionMantid_help"/>
    <addaction name="actionDNS_website"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_last">
   <property name="text">
    <string>Open last</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSimulation">
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionSingle_Crystal_Elastic">
   <property name="text">
    <string>Single Crystal Elastic</string>
   </property>
  </action>
  <action name="actionPowder_Elastic">
   <property name="text">
    <string>Powder Elastic</string>
   </property>
  </action>
  <action name="actionPowder_TOF">
   <property name="text">
    <string>Powder TOF</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionMantid_help">
   <property name="text">
    <string>Mantid help</string>
   </property>
  </action>
  <action name="actionDNS_website">
   <property name="text">
    <string>DNS website</string>
   </property>
  </action>
  <action name="actionSingle_Crystal_Elastic">
   <property name="text">
    <string>Single Crystal Elastic</string>
   </property>
  </action>
  <action name="actionSingle_Crystal_Elastic">
   <property name="text">
    <string>Single Crystal Elastic</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
